packages/angular/build/src/utils/supported-browsers.ts

```diff
@@ -8,14 +8,14 @@
 
 import browserslist from 'browserslist';
 
+const BASELINE_DATE = 'BASELINE-DATE-PLACEHOLDER';
```

Comment:
Suggestion: A comment that this will be replaced by `npm_package` might be helpful, otherwise this looks like dead code.